README.md

```diff
@@ -11,12 +11,9 @@ This repository contains a pipeline from taking bills from Massachusetts legisla
 * [EDA](https://github.com/vynpt/ml-maple-bill-summarization/tree/dev/EDA): the notebook ```eda.ipynb``` includes our work from scraping data that takes bills from MAPLE Swagger API, creating a dataframe to clean and process data, making visualizations to analyze data and explore characteristics of the dataset.
   
 * [demoapp](https://github.com/vynpt/ml-maple-bill-summarization/tree/dev/demoapp):   
-  ```app.py```: contains the codes of the LLM service we used and the wepapp we made using Streamlit. The webapp allows user to search for all bills.  
-  ```demo_app_with_12bills.py```: we test on top 12 bills from MAPLE website. We extract information from [Massachusetts General Law](https://malegislature.gov/Laws/GeneralLaws) to add context for the summaries of these bills. MGL sections text was scraped using extract_mgl_sections.py. Contains code to generate bill categories and tags
```

Comment:
The `demo_app_with_12bills` text should not be deleted - that app is still in use!

Reply:
fixed

Reply:
@azheng1991 are you sure you pushed this fix? I still see the text as being deleted for 12bills.

Reply:
I believe so...the changes I made are in the commit history and showing up on the branch at least:
<img width="833" alt="image" src="https://github.com/maple-testimony/ml-maple-bill-summarization/assets/58377006/ee50e597-91d7-4659-ab21-0dbe1a7dc0c4">

Reply:
Got it, I see it. Thanks!